### PR TITLE
Fix no owners for subfolders

### PR DESCRIPTION
**Pull request by myakove**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new pytest fixture for testing GitHub webhook processing.
  - Added classes for handling file trees and repository simulation in tests.
  - Implemented tests for evaluating pull request size thresholds based on additions and deletions.

- **Bug Fixes**
  - Enhanced logic for processing ownership data in GitHub webhook handling.

- **Refactor**
  - Updated type annotations for improved data structure representation.
  - Removed unnecessary classes from test files to streamline structure.
  - Adjusted coverage reporting settings for stricter requirements.

- **Documentation**
  - Configuration updates for pytest and coverage targets documented.

### CODE REVIEW

**Review by dbasunag — APPROVED** [review]

File: webhook_server_container/tests/test_pull_request_size.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class PullRequest:
+    def __init__(self, additions: int, deletions: int, labels: list[str] | None = None):
+        self.additions = additions
+        self.deletions = deletions
+        self.labels = labels or []
+
+    @property
+    def lables(self) -> list[Label]:
```
Comment (dbasunag) [inline webhook_server_container/tests/test_pull_request_size.py:18]: You missed this one.

Comment (myakove) [reply webhook_server_container/tests/test_pull_request_size.py:18]: Make it in your PR :)

**Pull request merged**